src/variables.ts

```diff
@@ -38,6 +38,7 @@ export const GRAPHIC_SECOND_10 = 'rgba(229, 229, 234, 0.82)';
 export const GRAPHIC_SECOND_11 = 'rgba(255, 255, 255, 0.2)';
 export const GRAPHIC_SECOND_12 = '#b2b2b2';
 export const GRAPHIC_SECOND_13 = 'rgba(0, 0, 0, 0.5)';
+export const GRAPHIC_SECOND_14 = 'rgba(0, 0, 0, 0)';
```

Comment:
you need to rename this variable to something more meaningful so that it does not repeat the name from the design